src/alfasim_sdk/_internal/alfacase/schema.py

```diff
@@ -560,6 +580,26 @@
         Optional("surface_tension"): Enum(['Baker Swerdloff']),
     }
 )
+pvt_model_table_parameters_description_schema = Map(
```

Comment:
is there a reason to not add `pt_` prefix?
```suggestion
pt_pvt_model_table_parameters_description_schema = Map(
```

Reply:
There is not a reason, since the old name definition was `pvt_model...` to indicate a `pt_pvt_model` I left it unchanged.

Reply:
> For example, the PvtModelTableParametersDescription could be PtPvtModelTableParametersDescription and so on.

I think it is great. It would let things more clear.

Reply:
> There is not a reason, since the old name definition was `pvt_model...` to indicate a `pt_pvt_model` I left it unchanged.

If you are referring to the general `PvtModel` I think it should keep the name, since it holds any kind of Pvt Models, Like PtTable, PhTable, Composition, ....

However, if it is related to the Original `PvtTable`, I think it should be changed to `PvtPtTable`.